src/views/login/index.js

```diff
@@ -0,0 +1,59 @@
+import React, { useCallback, useState } from 'react'
+
+import JungleIcon from '../../../public/assets/icons/jungle-devs-logo.svg'
+import Button from '../../components/button'
+
+import Input from '../../components/input'
+
+import styles from './styles.module.css'
+
+const Login = () => {
+  const [email, setEmail] = useState('')
+  const [password, setPassword] = useState('')
+  const [error, setError] = useState(false)
+
+  const onInputChange = useCallback((event, state) => {
+    const { value } = event.target
+
+    state(value)
+  }, [])
+
+  return (
+    <div className={styles['login']}>
+      <div className={styles['login-card']}>
+        <svg className={styles['login-icon']} viewBox={JungleIcon.viewBox} aria-hidden="true">
+          <use xlinkHref={`#${JungleIcon.id}`}/>
+        </svg>
+
+        <form className={styles['form-container']}>
+          <Input
+            id="email-input"
+            label="email"
+            hiddenLabel
+            value={email}
+            onChange={event => onInputChange(event, setEmail)}
```

Comment:
Avoid calling a function like this directly on onChange/onClick. This will lead to reconciliation issues.
What I'd suggest is either have 2 different callback functions to handle the input changes (one for email and another for password) or having a local reducer (useReducer) with all input values.